k256/src/arithmetic/projective.rs

```diff
@@ -250,6 +253,71 @@ impl From<AffinePoint> for ProjectivePoint {
     }
 }
 
+impl Normalize for ProjectivePoint {
```

Comment:
I know this code isn't DRY, but the call to invert_batch_generic in one case and invert_batch in the other made it hard to DRY-out. Suggestions are welcome

Reply:
I can think of some possibilities here but they might require some experimentation. This seems fine for now.

Reply:
This is what I had in mind, FWIW https://github.com/RustCrypto/traits/pull/1376/files#r1393273429